azure-pipelines.yml

```diff
@@ -21,6 +21,7 @@ jobs:
           # message is the second to last commit
           COMMIT_ID=$(echo $BUILD_SOURCEVERSIONMESSAGE | awk '{print $2}')
           message=$(git log $COMMIT_ID -1 --pretty=%B)
+          message=$(echo $message)
```

Comment:
Do we need this?

Reply:
Yes otherwise only the first line of the commit message is propagated